modules/tinymce/src/core/main/ts/api/Options.ts

```diff
@@ -260,13 +260,13 @@ const register = (editor: Editor): void => {
   registerOption('indent_before', {
     processor: 'string',
     default: 'p,h1,h2,h3,h4,h5,h6,blockquote,div,title,style,pre,script,td,th,ul,ol,li,dl,dt,dd,area,table,thead,' +
-      'tfoot,tbody,tr,section,summary,article,hgroup,aside,figure,figcaption,option,optgroup,datalist'
+      'tfoot,tbody,tr,section,details,summary,article,hgroup,aside,figure,figcaption,option,optgroup,datalist'
```

Comment:
Yes, it might affect contents. I will add that.

Reply:
I added two changelog items unsure under what category to add them added them under Improved. Since Added is more for new features etc. Maybe it should be under changed. But they are improvments.